tox.ini

```diff
@@ -41,4 +41,5 @@ commands = flake8 pylib
 # Ignore the following pep8 violations
 # E501: 80 character line length limit
 # E128: continuation line under-indented for visual indent
-ignore = E501, E128
\ No newline at end of file
+# W605: invalid escape sequence
+ignore = E501, E128, W605
```

Comment:
What's the reason for excluding this? Can we instead fix the offending lines or do a local ignore if fixing is not feasible..

Reply:
problem is ci env tools moved on, there is no  offending lines here.